xml/cap_admin_app_autoscaler.xml

```diff
@@ -446,12 +429,9 @@
 
   <para>
    For an example that demonstrates defining multiple scaling rules in a single
-   policy, refer to this
-   <link xlink:href="https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/src/integration/fakePolicyWithSchedule.json">sample
-   policy</link> file. The complete list of configurable policy values can be
+   policy, refer to this sample of a policy file <link xlink:href="https://github.com/cloudfoundry-incubator/app-autoscaler/blob/develop/src/integration/fakePolicyWithSchedule.json"/>. The complete list of configurable policy values can be
```

Comment:
With this change the output would like:

`For an example that demonstrates defining multiple scaling rules in a single policy, refer to this sample of a policy file <link>.`

The link could be better integrated with the rest of the sentence so that it doesn't look like a link is placed at the end of a completed sentence.

Some examples:
* `For an example that demonstrates defining multiple scaling rules in a single policy, refer to the sample policy file at <link>.`
* `Refer to <link> for an example policy file that demonstrates defining multiple scaling rules in a single policy`